src/popup/popup.js

```diff
@@ -297,6 +336,8 @@ async function nextRequest(page) {
 // global varialbe to check the status if user is viewwing the bookmarks section
 window.isBookmarksActive = false;
 
+elements.homeIcon.addEventListener('click', loadStoredSearch);
```

Comment:
Do not add a new event listener to the `homeIcon` as there is already one in `bookmarkModule.js` ([line 296](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/popup/bookmarkModule.js#L296)). Export the `loadStoredSearch` function and invoke it at the bottom of the evenlistener function.

Reply:
For that:
 > Should I re-write the function in bookmarkModule.js
 > Or I import popup.js in bookmarkModule.js and call that function
 > Or Should i create a new file for these storage functionalities ( It will refactor the code, so i think it's better).

@makkoncept Plz tell.

Reply:
For now, you can import `loadStoredSearch` function in `bookmarkModule.js` and invoke it at the end of the already present eventListener of homeIcon.